Integrate-Kobiton.md

```diff
@@ -0,0 +1,257 @@
+# Run Kobiton Automation Test on CircleCI
+## Table of contents
```

Comment:
The format looks wrong, is `Prerequisites` parent section or same section with `Configuring GitHub repository ...`